src/sqlite/persistence-strategy.ts

```diff
@@ -26,6 +27,13 @@ export class SqlitePersistenceStrategy implements PersistenceStrategy {
   private readonly eventMap: { [aggregateEvent: string]: IEventConstructor };
 
   constructor(private readonly options: SqliteOptions) {
+    if (options.connectionString === null
+      || options.connectionString === undefined) {
```

Comment:
wäre nicht `if (typeof options.connectionString !== 'string')`

kürzer und eindeutiger?

Reply:
Ich wollte keine objects mit ner expliziten toSting function ausschließen.
Aber ja, das würde funktionieren und is vll eindeutiger.

Reply:
naja andersrum erlaubst du so auch nummern etc, arrays etc